app/psc/rev-b.toml

```diff
@@ -0,0 +1,356 @@
+task-slots = ["sys"]
+
+[tasks.i2c_driver.interrupts]
+"i2c2.event" = 0b0000_0010
+"i2c2.error" = 0b0000_0010
+"i2c3.event" = 0b0000_0100
+"i2c3.error" = 0b0000_0100
+"i2c4.event" = 0b0000_1000
+"i2c4.error" = 0b0000_1000
+
+[tasks.update_server]
+name = "stm32h7-update-server"
+priority = 2
+max-sizes = {flash = 16384, ram = 4096}
+stacksize = 2048
+start = true
+uses = ["flash_controller", "bank2"]
+interrupts = {"flash_controller.irq" = 0b1}
+
+[tasks.hiffy]
+name = "task-hiffy"
+features = ["h753", "stm32h7", "itm", "i2c", "gpio", "spi"]
+priority = 3
+max-sizes = {flash = 32768, ram = 16384 }
+stacksize = 1024
+start = true
+task-slots = ["sys", "i2c_driver"]
+
+[tasks.net]
+name = "task-net"
+stacksize = 4640
+priority = 3
+features = ["mgmt", "h753", "vlan"]
+max-sizes = {flash = 131072, ram = 32768, sram1 = 16384}
+sections = {eth_bulk = "sram1"}
+uses = ["eth", "eth_dma", "system_flash"]
+start = true
+interrupts = {"eth.irq" = 0b1}
+task-slots = ["sys", { spi_driver = "spi2_driver" }]
+
+[tasks.mgmt_gateway]
+name = "task-mgmt-gateway"
+priority = 4
+max-sizes = {flash = 32768, ram = 8192}
+stacksize = 1536
+start = true
+uses = [
+    "system_flash", # TODO also used by `net`, both to read the stm32 uid
+]
+task-slots = ["jefe", "net", "update_server", "sys"]
+features = ["psc", "vlan"]
+
+[tasks.udpecho]
+name = "task-udpecho"
+priority = 4
+max-sizes = {flash = 16384, ram = 8192}
+stacksize = 4096
+start = true
+task-slots = ["net"]
+features = ["vlan"]
+
+[tasks.udpbroadcast]
+name = "task-udpbroadcast"
+priority = 4
+max-sizes = {flash = 16384, ram = 8192}
+stacksize = 4096
+start = true
+task-slots = ["net"]
+features = ["vlan"]
+
+[tasks.udprpc]
+name = "task-udprpc"
+priority = 4
+max-sizes = {flash = 32768, ram = 8192}
+stacksize = 4096
+start = true
+task-slots = ["net"]
+features = ["vlan"]
+
+[tasks.eeprom]
+name = "drv-eeprom"
+priority = 3
+max-sizes = {flash = 2048, ram = 256}
+stacksize = 256
+start = true
+task-slots = ["i2c_driver"]
+
+[tasks.idle]
+name = "task-idle"
+priority = 5
+max-sizes = {flash = 128, ram = 256}
+stacksize = 256
+start = true
+
+[config]
+#
+# I2C2: Local bus
+#
+[[config.i2c.controllers]]
+controller = 2
+
+#
+# I2C_SP_TO_LOCAL_SDA
+# I2C_SP_TO_LOCAL_SCL
+#
+[config.i2c.controllers.ports.F]
+name = "local"
+description = "Local bus"
+pins = [ { pins = [ 0, 1 ], af = 4 } ]
+
+#
+# I2C3: Backplane bus
+#
+[[config.i2c.controllers]]
+controller = 3
+
+#
+# I2C_SP_TO_PS_SCL
+# I2C_SP_TO_PS_SDA
+#
+[config.i2c.controllers.ports.H]
+name = "backplane"
+description = "Backplane bus"
+pins = [ { pins = [ 7, 8 ], af = 4 } ]
+
+[[config.i2c.devices]]
+bus = "local"
+address = 0b1001_000
+device = "tmp116"
+description = "Temperature sensor"
+
+[[config.i2c.devices]]
+bus = "local"
+address = 0b1010_000
+device = "at24csw080"
+description = "FRU ID EEPROM"
+
+[[config.i2c.devices]]
+bus = "backplane"
+name = "psu1eeprom"
+address = 0b1010_000
+device = "psueeprom"
+description = "PSU 1 EEPROM"
```

Comment:
Per RFD 200, rectifiers should be labeled starting at 0. I think we want to use our labeling as opposed to Murata's doc-only 1 based.

Reply:
There are other places where schematics and related diverge from what we ultimately settled on for nomenclature. I'm not sure if it's worth the churn in the Rev C schematic or not, I'll leave that up to folks. I think the important thing is that when the SP software reports up to the control plane we use the actual identifiers that we're labeling the system with and following those conventions. If that means we have a delta from the schematic to that (which we have in Gimlet due to changes in how we were labeling PCIe slots), then that's something we can probably live with.

Reply:
Okay, I've updated both rev A and B files to use zero-indexing, with a note explaining that the schematic is different.